CONTRIBUTING.md

```diff
@@ -37,22 +52,23 @@ poetry shell
 
 To connect virtual environment interpreter to IDE the `which python` command can be useful.
 
-### 3. Suggesting a feature (optional)
-
-This is an optional step, you could skip it if you want to take some of the existing tasks.
+### 3. Choosing a task
 
-1. Make an issue with your feature description;
-2. We shall discuss the design and its implementation details;
-3. Once we agree that the plan looks good, go ahead and implement it.
+Ready to do tasks are present at [GitHub Issues](https://github.com/etna-team/etna/issues):
+- Pick an issue with status "Todo" on a [board](https://github.com/orgs/etna-team/projects/1);
```

Comment:
Do all the people have access to this board?

Reply:
It opens in incognito tab without any github account. But I agree that we should clarify what the work "pick" mean.